src/draw/lv_draw_vector.c

```diff
@@ -325,7 +325,15 @@ void lv_vector_path_close(lv_vector_path_t * path)
 
 void lv_vector_path_get_bounding(const lv_vector_path_t * path, lv_area_t * area)
 {
+    if(path == NULL || area == NULL) {
```

Comment:
Should we assert here or leave the error to caller?

Reply:
LV_ASSERT_NULL(path);
LV_ASSERT_NULL(area);